cypress-tests/cypress/e2e/PaymentUtils/WorldPay.js

```diff
@@ -0,0 +1,508 @@
+        },
+      }
+    },
+    // SaveCardUseNo3DSAutoCaptureOffSession: {
+    //     Request: {
+    //         payment_method: "card",
+    //         payment_method_data: {
+    //             card: successfulNo3DSCardDetails,
+    //         },
+    //         setup_future_usage: "off_session",
+    //         customer_acceptance: {
+    //             acceptance_type: "offline",
+    //             accepted_at: "1963-05-03T04:07:52.723Z",
+    //             online: {
+    //                 ip_address: "127.0.0.1",
+    //                 user_agent: "amet irure esse",
+    //             },
+    //         },
+    //     },
+    //     Response: {
+    //         status: 200,
+    //         body: {
+    //             status: "processing",
+    //         },
+    //     },
+    // },
+    // SaveCardConfirmAutoCaptureOffSession: {
+    //     Request: {
+    //         setup_future_usage: "off_session",
+    //     },
+    //     Response: {
+    //         status: 400,
+    //         body: {
+    //             type: "invalid_request",
+    //             message: "Missing required param: payment_method_data",
+    //             code: "IR_19",
+    //         },
+    //     },
+    // },
+    // SaveCardConfirmManualCaptureOffSession: {
+    //     Request: {
+    //         setup_future_usage: "off_session",
+    //     },
+    //     Response: {
+    //         status: 400,
+    //         body: {
+    //             error: {
+    //                 type: "invalid_request",
+    //                 message: "Payment method type not supported",
+    //                 code: "IR_19",
+    //                 reason: "debit mandate payment is not supported by worldpay"
+    //             }
+    //         },
+    //     },
+    // },
+
+    /**
+     * Variation cases
+     */
+    CaptureCapturedAmount: {
+      Request: {
+        Request: {
+          payment_method: "card",
+          payment_method_data: {
+            card: successfulNo3DSCardDetails,
+          },
+          currency: "EUR",
+          customer_acceptance: null,
+        },
+      },
+      Response: {
+        status: 400,
+        body: {
+          error: {
+            type: "invalid_request",
+            message:
+              "This Payment could not be captured because it has a capture_method of automatic. The expected state is manual_multiple",
+            code: "IR_14",
+          },
+        },
+      },
+    },
+    ConfirmSuccessfulPayment: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "USD",
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 400,
+        body: {
+          error: {
+            type: "invalid_request",
+            message:
+              "You cannot confirm this payment because it has status processing",
+            code: "IR_16",
+          },
+        },
+      },
+    },
```

Comment:
these are handled by `commons.js`?

Reply:
They're present for a different status (`succeeded`)